packages/runtime/src/widgets/Form/TextInput.tsx

```diff
@@ -15,7 +15,7 @@ export const TextInput: React.FC<TextInputProps> = (props) => {
   return (
     <AntForm.Item
       label={values.label}
-      name={values.label}
+      name={values.id}
```

Comment:
This should use the provided ID if it exists (not the resolved one), and then fallback to the label

Reply:
so are you suggesting to use label then @evshi